extensions/amp-access/0.1/amp-access.js

```diff
@@ -149,6 +166,9 @@ export class AccessService {
 
     // Start authorization XHR immediately.
     this.runAuthorization_();
+
+    // Wait for the "view" signal.
+    this.scheduleView_();
```

Comment:
Oh. I see your point. This makes sense. We definitely need to try again if the view has not been reported yet. Will do.